cmd/thanos/query.go

```diff
@@ -163,6 +165,16 @@ func registerQuery(app *extkingpin.App) {
 			return errors.Wrap(err, "parse federation labels")
 		}
 
+		var enableNegativeOffset, enableAtModifier bool
+		for _, feature := range *featureList {
+			if feature == "promql-negative-offset" {
```

Comment:
Let's use constants, otherwise it's easy for typos

Reply:
We can then render those constants in flag help 👍🏽

Reply:
@bwplotka thanks for the review. Could you please check it now and let me know if I coded it right?

Reply:
I have squashed all the commits into one for a cleaner commit history.